### PR TITLE
Add Russian translation

### PR DESCRIPTION
**Pull request by JonnyBro**

### CODE REVIEW

File: src/main/resources/assets/fuelinfo/lang/ru_ru.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "message.fuelinfo.furnace.items": "§7Эта печь может переплавить §e%d §7предмет(а/ов)",
+  "message.fuelinfo.furnace.stacks": "§7Эта печь может переплавить §e%d §7стак(а/ов)",
+  "message.fuelinfo.furnace.both": "§7Эта печь может переплавить §e%d §7стак(а/ов) и §e%d §7предмет(а/ов)",
+
+  "message.fuelinfo.brewing_stand": "§7This Brewing Stand can produce §e%d §7potions"
```
Comment (Luligabi1) [inline src/main/resources/assets/fuelinfo/lang/ru_ru.json]: Can you translate this line as well? 🙂

Comment (JonnyBro) [reply src/main/resources/assets/fuelinfo/lang/ru_ru.json]: I'm at work right now, will do it later. Don't know how I missed this line 😄

Comment (JonnyBro) [reply src/main/resources/assets/fuelinfo/lang/ru_ru.json]: Fixed it, sorry!

Comment (Luligabi1) [reply src/main/resources/assets/fuelinfo/lang/ru_ru.json]: No problem, thank you for your contribution ^^

**Review by Luligabi1 — APPROVED** [review]

**Pull request merged**